### ISSUE TITLE
idevicesyslog, doesn't show debug/info messages in output

### ISSUE THREAD

**Issue by 0x3c3e** [libimobiledevice/libimobiledevice#1141]
`os_log("MSG", type: .debug)` or `os_log("MSG", type: .info)` messages aren't visible in output even with `-d` flag.

Are there options like in Console.app to enable debug/info levels?

**Comment by francybiga** [comment]
It would be very useful to access debug and info logs!

**Comment by jay0129** [comment]
I need this feature so badly!

**Comment by localip** [comment]
Bump, is this planned?

**Comment by localip** [comment]
@libimobiledevice

**Comment by ecunha-ta** [comment]
Any news on this?

**Comment by sejinik** [comment]
is this planned?!  it will be helpful to user

**Comment by lucasfernog** [comment]
I wish this had a solution too :(

**Comment by Dev1an** [comment]
Yes I tried idevicesyslog but that only shows messages of type .default, .error or .fault
Is idevicesyslog part of libimobildevice or is there a separate repo for that?

**Comment by Dev1an** [comment]
@nikias What is the "new updated idevicesyslog"? I tried `idevicesyslog 1.3.0`

**Comment by nikias** [comment]
You need to compile it from source.
